src/model/element-base.ts

```diff
@@ -107,4 +120,4 @@ export abstract class ElementBase implements Feature {
   emitEventMetadata(_event: Event): Object {
     return {};
   }
-}
+}
```

Comment:
Done.  Also, updated vscode settings.json for project to do that https://github.com/Polymer/polymer-analyzer/pull/599